### DISCUSSION TITLE
Help Center Section parent id attribute

### DISCUSSION THREAD

**Discussion by pmorddny** [cloudbees/zendesk-java-client#514] [Ideas]
On the API documentation https://developer.zendesk.com/api-reference/help_center/help-center-api/sections/ there's a parent_section_id attribute mentioned. I don't see it on the Section class. Could it be added when you get a chance?

thanks!

**Comment by PierreBtz** [comment]
@pmorddny hello, sorry for the delay, I created an issue out of this.
